testers/testers/sql/bin/install.sh

```diff
@@ -7,16 +7,30 @@ install_packages() {
     sudo apt-get install python3 postgresql
 }
 
+random_pwd() {
+    date +%s%N | sha256sum | base64 | head -c 32
+}
+
 create_db_and_users() {
     echo "[SQL-INSTALL] Creating databases and users"
     echo "[SQL-INSTALL] Creating oracle user '${ORACLEUSER}' with database '${ORACLEDB}'"
     sudo -u postgres psql <<-EOF
 		DROP DATABASE IF EXISTS ${ORACLEDB};
 		DROP ROLE IF EXISTS ${ORACLEUSER};
 	EOF
```

Comment:
I would drop this, since we now have an unistallation script, and we probably want it to fail here if there are leftovers (or a db with the exact names we want).